experimental/handoff-library/csharp_dotnetcore/samples/LivePersonAgentBot/EchoBot.runtimeconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "runtimeOptions": {
+    "tfm": "netcoreapp2.1",
```

Comment:
I would *strongly* consider replatforming this to Net Core 3.1. Net core 2.1 is already in "Long Term Support" and is really not recommended for new projects at this point. 
https://dotnet.microsoft.com/platform/support/policy/dotnet-core